walkthroughs/eks/o11y-prometheus.md

```diff
@@ -56,6 +81,34 @@ Similarly, you can see all the scraped metrics (including application health met
 
 ## Cleanup
 
+helm3 would support --purge by default
+```
+helm delete appmesh-prometheus --namespace appmesh-system
+```
+
+## Troubleshooting
+
+If the Prometheus port does not open properly, first determine if the Pod is functioning properly.  
+```
+kubectl -n appmesh-system get deploy,po,svc
+NAME                                 READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/appmesh-controller   1/1     1            1           42m
+deployment.apps/appmesh-prometheus   0/1     1            0           33m
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+pod/appmesh-controller-6dcf8c7787-zgh7w   1/1     Running   0          42m
+pod/appmesh-prometheus-6d6ffbb888-5644r   0/1     Pending   0          30m
+
+NAME                                         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+service/appmesh-controller-webhook-service   ClusterIP   10.100.96.3     <none>        443/TCP    42m
+service/appmesh-prometheus                   ClusterIP   10.100.53.248   <none>        9090/TCP   33m
+```
+If the Pod status is unhealthy, first check the status of the PVC, and check the log in event:  
+```
+kubectl describe pvc -n appmesh-system
+```
+If the problem is not solved, check the node's resource deployment, memory and CPU limits: 
 ```
-helm delete appmesh-prometheus -n appmesh-system
+kubectl describe nodes
 ```
+If the node doesn't have enough resources, you can scale
```

Comment:
Can the be extended a bit? Maybe `If the node doesn't have enough resources, you can try scaling the cluster.`

Reply:
will do